macos_instructions.yml

```diff
@@ -0,0 +1,132 @@
+# actionOnlyInformation contains information necessary for the GitHub action to run,
+# but not useful to an end user
+
+# setupInformation contains two different keys,
+# 1. Text
+#   This is displayed only in README.md and is pasted exactly into README.md, this means
+#   that any md formatting must be made to the text in the text field.
+#   For example:
+#   text: ### This is a heading
+#
+# 2. Code
+#   The key run is the only field pasted into README.md. All the fields are added
+#   to the end of the steps in macos_installation_instruction.yml.
+#   NOTE: Do not have multiline code, this will cause problems for github actions, instead use ';'s
+#   scripts/macos-instructions.sh will substitute each ';' for a new line in README.md.
+#   NOTE: "|| true" will be removed from the code shown on README.md
+
+actionOnlyInformation:
+    name: Verify MacOS Installation Instructions
+    # The file macos_installation_instructions.yml is generated with the script scripts/macos-instructions.sh
+    on:
+      schedule:
+        - cron: '30 12 * * 1'  #  This is every Monday at 5:30am PST/8:30am EST
+    jobs:
+      build:
+        runs-on: macos-latest
+        steps:
+          - name: Checkout dockstore/docktore
+            uses: actions/checkout@v3
+            with:
+              repository: dockstore/dockstore
+              path: dockstore
+          - name: Checkout dockstore/docktore-ui2
+            uses: actions/checkout@v3
+            with:
+              repository: dockstore/dockstore-ui2
+              path: dockstore-ui2
+          - name: Setup Java
+            uses: actions/setup-java@v3
+            with:
+              distribution: 'temurin'
+              java-version: '17'
+setupInformation:
+    - text: '### Setting up a Mac for Dockstore development'
+    - text: Install Docker (Be sure to click on 'Mac with Apple chip' if you have Apple silicon)
+    - text: https://docs.docker.com/desktop/mac/install/
+    - newLine: 1
+    - text: Install Brew
+    - text: https://brew.sh/
+    - code:
+        run: /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+        working-directory: dockstore
```

Comment:
Does this need to be set here and the other places? What if the directory doesn't exist?

And what does it actually do? I don't see it in the REAMDE.md?

Reply:
This information is only for the github action workflow. 

It is the directory where the repo was cloned. 

I have removed ` working-directory: dockstore` from everything except for the instructions that require the use of the cloned repo.

[29c5c17](https://github.com/dockstore/dockstore/pull/5043/commits/29c5c17e6e19e092614c925368db8eb0bb035a85)